test/controllers/group_pair_controller_test.exs

```diff
@@ -0,0 +1,101 @@
+defmodule Pairmotron.GroupPairControllerTest do
+  use Pairmotron.ConnCase
+
+  import Pairmotron.TestHelper,
+    only: [log_in: 2, create_retro: 2, create_pair_and_retro: 2]
+
+  describe "while authenticated" do
+    setup do
+      user = insert(:user)
+      group = insert(:group, %{owner: user, users: [user]})
+      conn = build_conn() |> log_in(user)
+      {:ok, [conn: conn, logged_in_user: user, group: group]}
+    end
+
+    test "lists one active user", %{conn: conn, logged_in_user: user, group: group} do
+      conn = get(conn, "/groups/#{group.id}/pairs")
+      assert html_response(conn, 200) =~ user.name
+    end
+
```

Comment:
I feel like this is handled by the tests of non-group membership access and inactive users.